packages/invoices/src/OutboundShipment/DetailView/modals/hooks/usePackSizeController.tsx

```diff
@@ -0,0 +1,87 @@
+import { useState, useEffect, useMemo } from 'react';
```

Comment:
Also very little changed here a part from extracting the hook into it's own file & also extracting some of the code within the hook to their own methods - primarily for my own understanding as I was getting a little confused 😁